akka-actor-typed/src/main/scala/akka/actor/typed/internal/adapter/ActorSystemAdapter.scala

```diff
@@ -58,7 +59,7 @@ import akka.event.LoggingFilterWithMarker
   def isTerminated: Boolean = whenTerminated.isCompleted
 
   final override val path
-      : untyped.ActorPath = untyped.RootActorPath(untyped.Address("akka", untypedSystem.name)) / "user"
+    : untyped.ActorPath = untyped.RootActorPath(untyped.Address("akka", untypedSystem.name)) / "user"
```

Comment:
Now I branched from master today, and format-on-save still formatted it this way!

Reply:
For med it formats as it was (when I edit and save)
```
  final override val path
      : untyped.ActorPath = untyped.RootActorPath(untyped.Address("akka", untypedSystem.name)) / "user"
```

Not sure if the version of the plugin is visible somewhere but this is the content I have:
```
$ ll /Applications/IntelliJ\ IDEA\ CE.app/Contents/plugins/intellij-scalafmt/lib
total 41088
drwxr-xr-x@ 32 patrik  staff     1024 Mar  6 14:01 .
drwxr-xr-x@  4 patrik  staff      128 Mar 11 11:48 ..
-rwxr-xr-x@  1 patrik  staff   812555 Sep 19  2018 common_2.12-4.0.0.jar
-rwxr-xr-x@  1 patrik  staff   219554 May  2  2014 config-1.2.1.jar
-rwxr-xr-x@  1 patrik  staff   286090 Feb 21  2018 config-1.3.3.jar
-rwxr-xr-x@  1 patrik  staff  1750498 Nov 26 14:06 coursier-small_2.12-1.3.1.jar
-rwxr-xr-x@  1 patrik  staff    25442 Sep 19  2018 dialects_2.12-4.0.0.jar
-rwxr-xr-x@  1 patrik  staff    34130 Mar  4  2013 diffutils-1.3.0.jar
-rwxr-xr-x@  1 patrik  staff    64299 Sep 12  2017 fansi_2.12-0.2.5.jar
-rwxr-xr-x@  1 patrik  staff    69262 Oct 14  2017 fastparse-utils_2.12-1.0.0.jar
-rwxr-xr-x@  1 patrik  staff   327502 Oct 14  2017 fastparse_2.12-1.0.0.jar
-rwxr-xr-x@  1 patrik  staff    51723 Sep 19  2018 inputs_2.12-4.0.0.jar
-rwxr-xr-x@  1 patrik  staff  1304653 Mar  6 14:01 intellij-scalafmt.jar
-rwxr-xr-x@  1 patrik  staff    65132 Sep 19  2018 io_2.12-4.0.0.jar
-rwxr-xr-x@  1 patrik  staff   318230 May 29  2018 metaconfig-core_2.12-0.8.3.jar
-rwxr-xr-x@  1 patrik  staff    10439 May 29  2018 metaconfig-typesafe-config_2.12-0.8.3.jar
-rwxr-xr-x@  1 patrik  staff    32927 Dec  3  2012 nailgun-server-0.9.1.jar
-rwxr-xr-x@  1 patrik  staff    69472 Jun 23  2017 paiges-core_2.12-0.2.0.jar
-rwxr-xr-x@  1 patrik  staff   693539 Sep 19  2018 parsers_2.12-4.0.0.jar
-rwxr-xr-x@  1 patrik  staff   150484 Sep 12  2017 pprint_2.12-0.5.3.jar
-rwxr-xr-x@  1 patrik  staff   178288 Sep 19  2018 quasiquotes_2.12-4.0.0.jar
-rwxr-xr-x@  1 patrik  staff  5272591 Dec  4 10:15 scala-library-2.12.8.jar
-rwxr-xr-x@  1 patrik  staff  3645095 Dec  4 10:14 scala-reflect-2.12.8.jar
-rwxr-xr-x@  1 patrik  staff   553347 Sep 26 19:32 scala-xml_2.12-1.1.1.jar
-rwxr-xr-x@  1 patrik  staff    15757 Sep 19  2018 scalameta_2.12-4.0.0.jar
-rwxr-xr-x@  1 patrik  staff    73516 Nov  2  2016 scopt_2.12-3.5.0.jar
-rwxr-xr-x@  1 patrik  staff  1549025 Sep 19  2018 semanticdb_2.12-4.0.0.jar
-rwxr-xr-x@  1 patrik  staff   110958 Jul  9  2017 sourcecode_2.12-0.1.4.jar
-rwxr-xr-x@  1 patrik  staff   124242 Sep 19  2018 tokenizers_2.12-4.0.0.jar
-rwxr-xr-x@  1 patrik  staff   512751 Sep 19  2018 tokens_2.12-4.0.0.jar
-rwxr-xr-x@  1 patrik  staff    93783 Sep 19  2018 transversers_2.12-4.0.0.jar
-rwxr-xr-x@  1 patrik  staff  2564599 Sep 19  2018 trees_2.12-4.0.0.jar
```